### PR TITLE
reduce string literals for settings with const helper

### PR DESCRIPTION
**Pull request by lwhiteley**
Changes: 
- reduce repetitive string literals and have central place for holding keys
- Extra: format start log in console

console start

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -6,13 +6,23 @@ import app from './lib/server';
 import config from './config';
 import logger from './lib/server/logger';
 import migrate from './lib/server/migrate';
+import {padStart} from 'lodash';
```
Comment (bruno12mota) [inline app.js]: Use lodash cherry pick:

```js
import padStart from 'lodash/padStart';
```

File: lib/server/index.js [hunk 2]
```diff
@@ -11,6 +11,14 @@ import passport from 'passport';
...
 import safeHtmlString from 'helpers/safe-html-string';
 import session from 'express-session';
+import {
+  favicon,
+  title,
+  description,
+  keywords,
+  copyright,
+  robots
+} from 'helpers/const/setting-keys';
```
Comment (bruno12mota) [inline lib/server/index.js]: Place `setting-keys` in statics folder https://github.com/relax/relax/tree/master/lib/shared/statics and rename to `settings-keys`

File: lib/shared/screens/admin/shared/helpers/map-settings-ids.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
+import {map, compact} from 'lodash';
```
Comment (bruno12mota) [inline lib/shared/screens/admin/shared/helpers/map-settings-ids.js]: Cherry pick as well :)

File: lib/shared/screens/admin/shared/helpers/map-settings-ids.js [hunk 4]
```diff
@@ -0,0 +1,3 @@
...
+
+export default (options) => compact(map(options, (option) => option.id));
```
Comment (bruno12mota) [inline lib/shared/screens/admin/shared/helpers/map-settings-ids.js]: This implementation is quite weak. Options lists might have deep options on properties `unlocks` and `options`. Also you can use native `Array.prototype.map`

Comment (lwhiteley) [inline lib/shared/screens/admin/shared/helpers/map-settings-ids.js]: cool didnt know about the deeper props. will update this shortly

Comment (lwhiteley) [inline lib/shared/screens/admin/shared/helpers/map-settings-ids.js]: is there an example data structure you could point me to for  both unlocks and options properties that i can test against

Comment (bruno12mota) [inline lib/shared/screens/admin/shared/helpers/map-settings-ids.js]: Check here https://github.com/relax/relax/blob/master/lib/shared/elements/mobile-menu/style.js#L32 and https://github.com/relax/relax/blob/master/lib/shared/elements/mobile-menu/style.js#L49

Being that options is always an array and unlocks can be an array or an object.

Comment (lwhiteley) [inline lib/shared/screens/admin/shared/helpers/map-settings-ids.js]: great thanks. made the updates, let me know if any changes required

**Comment by bruno12mota** [comment]
Awesome PR mate, thanks! 😄

**Pull request merged**